configurations/init_test_workspace.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from core_data_modules.cleaners import swahili
+from core_data_modules.data_models import CodeScheme
+from temba_client.v2 import Field
+
+from src.common.configuration import RapidProClientConfiguration, CodaClientConfiguration, UUIDTableClientConfiguration, \
+    EngagementDatabaseClientConfiguration
+from src.engagement_db_to_coda.configuration import CodaSyncConfiguration, CodaDatasetConfiguration, \
```

Comment:
We should clean up unused imports here and add them when needed.

Reply:
Will address this shortly in a new pr